src/musicbrainz/web/acoustidlookuptask.cpp

```diff
@@ -81,6 +81,7 @@ AcoustIdLookupTask::AcoustIdLookupTask(
 QNetworkReply* AcoustIdLookupTask::sendNetworkRequest(
         QNetworkAccessManager* networkAccessManager,
         network::HttpRequestMethod method,
+        int parentTimeoutMillis,
```

Comment:
Unused?

Reply:
Yeah, I had a problem with this parameter. 

Normally we didn't need it in the first place, but since we are looking for all the possible cover arts we must send requests in sequences and it is needed to send it over and over. I added this parameter for only `CoverArtArchiveTask` possible new name is `CoverArtArchiveLinksTask` it was giving error due to inheritance (It is added just for coverartarchivelinkstask, and base class didn't have it). I couldn't find a proper solution to avoid this errors. Is there a better way to fix this errors?